pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1328,6 +1349,31 @@ func TestSchedulerCache_UpdateNodeInfoSnapshot(t *testing.T) {
 			},
 			expected: []*v1.Node{nodes[2], nodes[0]},
 		},
+		{
+			name: "Add Pods with affinity",
+			operations: []operation{
+				addNode(0), addPod(10), updateSnapshot(), addNode(1),
```

Comment:
To make it more readable, I suggest to add a new operation: `addPodWithAffinity`

Reply:
done